en/contributing/documentation.rst

```diff
@@ -15,7 +20,67 @@ Email the docs team (docs at cakephp dot org) or hop on IRC
 (#cakephp on freenode) to discuss any translation efforts you would
 like to participate in.
 
-Translator tips:
+New Translation Language
+------------------------
+
+We want to provide translations that are as complete as possible. However, there
+may be times where a translation file is not up-to-date. You should always
+consider the English version as the authoritative version.
+
+If your language is not in the current languages, please contact us through
+Github and we will consider creating a skeleton folder for it. The following
+sections are the first one you should consider translating as these
+files don't change often:
+
+- index.rst
+- intro.rst
+- quickstart.rst
+- installation.rst
+- /intro folder
```

Comment:
I can take care of this.
Would this be ok to you ?
- index.rst
- `/cakephp-overview` folder
- `/getting-started` folder
- `/installation` folder
- `/tutorials-and-examples` folder